app/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@storybook/addon-links": "^6.0.0",
     "@trussworks/react-uswds": "^5.0.0",
+    "@types/node": "^20.11.24",
```

Comment:
Needed so that TS can stop complaining about accessing the node process `process.env['SOME_VAR]`. Accessing the system environments can be useful for debugging, service setup, etc.